File: C/fibonacci_em_vetor.cpp
#include <stdio.h>
#include <stdlib.h>

int main() {
    int t;
    scanf("%d", &t);

    for (int a = 0; a < t; a++) {
        long int x;
        scanf("%d", &x);

        x = abs(x);
        double n1 = 0.0, n2 = 1.0;
		double soma = 0.0;

        if (x == 0) {
            printf("Fib(0) = 0\n");
        } else if (x == 1) {
            printf("Fib(1) = 1\n");
        } else if (x > 1) {
            for (int b = 2; b <= x; b++) {
                soma = n1 + n2;
                n1 = n2;
                n2 = soma;
            }
            printf("Fib(%d) = %.0lf\n", x, soma);
        }
    }

    return 0;
}

File: Python/fibonacci_em_vetor.py
t=int(input())

for a in range(0,t,1):
    x=int(input())
    x = abs(x)
    n1 = 0
    n2 = 1
    soma= 0
    if x == 0:
        print("Fib(0) = 0")
    elif x == 1:
        print("Fib(1) = 1")
    elif x > 1:
        for b in range(2,x+1,1):
            soma = n1 + n2
            n1 = n2
            n2 = soma
        print(f"Fib({x}) = {soma}")
